app/src/main/java/org/schabi/newpipe/player/event/PlayerEventListener.java

```diff
@@ -10,4 +11,6 @@
     void onProgressUpdate(int currentProgress, int duration, int bufferPercent);
     void onMetadataUpdate(StreamInfo info);
     void onServiceStopped();
+    void onFullScreenButtonClicked(boolean fullscreen);
```

Comment:
fragmentListener should be a class that extends PlayerEventListener, not adding callbacks to the existing listener class, which only emits passive events.